src/site/content/en/learn/pwa/assets-and-data/index.md

```diff
@@ -1,11 +1,140 @@
+Asset caching happens independently from PWA installation, 
+and will work even from a browser tab, allowing for faster asset delivery and an offline experience regardless of how a user chooses to use your PWA.
+{% endAside %}
+
+### Frequently used cache approaches
+
+In your PWA, you are in charge of all the decisions. You can choose the best approach to cache or install assets based on your needs.
+Some decisions to make are:
+
+* Precaching: pick the assets you'd like to install on the first load; those should include the HTML and the minimal assets to render the app. When using precache keep in mind you are using user device's space and network,be conscious and responsible when downloading assets and caching them. 
+* Cache as needed: used to add assets to the cache when they are requested. Typical these are assets that can change independently of your app version, like images or content. See the [caching](learn/pwa/caching) section for different strategies on how to cache as needed.
+* APIs and web services: API calls can be cached; or, instead of caching the API responses, you can store their data in IndexedDB.  
+
+{% Aside 'gotchas' %}
+With platform-specific apps, the most common pattern is to include all the app's assets in its package; that's why it's common these apps are larger than 500MB. On the other hand, A PWA can initially install only the minimum set of resources to offer an offline and fast experience and download the rest later, with PWAs using considerably less storage on installation. 
+{% endAside %}
+
+### Updating assets
+In the standard app model for app catalog installed apps, when developers need to update their app, they publish a new package. Users need to download the whole package again, even if most of the assets haven't changed.
+With PWAs, using the approaches on the section above, you decide how and when to update assets. 
+Here are different options for how to update assets:
+
+* Full Update: in this case, every time you make a change in the app, even a minor one, your code will download all the assets again in the cache.
+* Partial Update: in this approach, you create a method to define which assets have been updated, and you update only those, with or without the user's intervention.
+* Continuous Update: using this technique, your assets will be checked and updated automatically on every PWA usage individually
+
+{% Aside %}
+When caching assets, you need to decide on a cache swap or clean up policy that defines when the user will see an updated asset; taking into account storage usage and the balance between a fast experience and displaying a slightly stale asset.
+{% endAside %}
+
+## Size and Lifespan
+
+Usually platform-specific apps don't deal with size limits; at installation, apps can be gigabytes or more in size, as long as the device has the capacity the installation will be allowed. Also, while the app is installed, it will use that total amount of storage no matter if you use the app or not.
```

Comment:
```suggestion
Usually platform-specific apps don't deal with size limits; at installation, apps can be gigabytes or more in size. As long as the device has the capacity the installation will be allowed. Also, while the app is installed, it will use that total amount of storage no matter if you use the app or not.
```